thirdparty/nlohmann_json/cmake/download_test_data.cmake

```diff
@@ -0,0 +1,56 @@
+set(JSON_TEST_DATA_URL     https://github.com/nlohmann/json_test_data)
```

Comment:
Is this file really needed? I think we are not shipping the tests anyway.

Reply:
I just copied the directory whithout thinking much about it but you are right I should remove probably this. Let me check